rsync_backup.sh

```diff
@@ -221,46 +229,22 @@ MENU
         ;;
       3)
         echo "YOU CHOSE: 3. Backup Black USB -> Red Hard Drive"
-        # If the function call is successful, it continues with next line.
-        # If the function call is UNsuccessful, the function already gave user
-        # an invalid directory message & quit the process, so you have to choose
-        # a menu item again.
         echo "$src_is" "$source_path_black_usb"
         echo "$dest_is" "$dest_red"
         echo
-
         check_if_directory "$source_path_black_usb"
         check_if_directory "$dest_red"
-
```

Comment:
You could also move these into `do_backup_for_1_target`. I think you can get it to the point that each case looks like

```
echo "YOU CHOSE: ..."
do_backup_for_1_target ...
```

Reply:
@kcharter I see what you mean. Thanks.

Reply:
Looks good!